docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
`null`: Value of a property with no value
`undefined`: Value of a property that does not exist

Maybe `null` makes more sense. It indicates an `Unknown` value rather than nonexistent property.
:+1: for `null` @quangbuule thoughts?